deployment/local-jvm/README.md

```diff
@@ -29,6 +29,13 @@ To run the example locally in JVMs, follow these steps (running commands from th
 
```

Comment:
How about moving the line "To run the example ..." down to before the header Running using the Bash Scripts.